impl/opengl/opengl.odin

```diff
@@ -249,7 +249,7 @@ setup_imgui_shaders :: proc() -> u32 {
     
     ok: i32;
     gl.GetProgramiv(program_h, gl.LINK_STATUS, &ok);
-    if ok != gl.TRUE {
+    if ok != 1 {
```

Comment:
Why not `gl.TRUE`? Other places like line 195 uses it?

Reply:
In shared:opengl, TRUE was 1 and not a boolean. In vendor:opengl, TRUE is true and a boolean. I can cast ok to a bool and compare it that way.